app/views/shared/dialogs/_dialog_provision.html.haml

```diff
@@ -12,43 +12,15 @@
   - resource_action_id ||= @dialog_locals[:resource_action_id]
   - open_url ||= @dialog_locals[:open_url]
 
-- if Settings.product.old_dialog_user_ui
```

Comment:
I remembered that this branch was used somewhere in provisioning, however I couldn't reproduce it, so maybe my memory is just tricking me.

Assuming that this branch is not used anymore, first line `- wf = @edit[:wf] if @edit && @edit[:wf]` could be also deleted. 

Originally i thought that because of this cleanup we might be able to get rid of some parts of `miq_request_methods.rb` but in `_prov_wf.html.haml` we are still using wf dialogs.